doc_source/calico.md

```diff
@@ -2,12 +2,20 @@
 
 [Project Calico](https://www.projectcalico.org/) is a network policy engine for Kubernetes\. With Calico network policy enforcement, you can implement network segmentation and tenant isolation, which useful in multi\-tenant environments where you need to isolate tenants from each other or when you want to create separate environments for development, staging, and production\. Network policies are similar to AWS security groups in that you can create network ingress and egress rules, but instead of assigning instances to a security group, you assign network policies to pods using pod selectors and labels\. The following procedure shows you how to install Calico on your Amazon EKS cluster\. 
 
-**To install Calico on your Amazon EKS cluster**
+**To install or upgrade Calico on your Amazon EKS cluster**
 
-1. Apply the Calico manifest from the [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\. This manifest creates daemon sets in the `kube-system` namespace\.
+1. Apply the Amazon CNI manifest [`aws/amazon-vpc-cni-k8s` GitHub project](https://github.com/aws/amazon-vpc-cni-k8s)\.
```

Comment:
Only users upgrading from https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/v1.0.0/config/v1.0/aws-k8s-cni-calico.yaml need to run this step.

Reply:
I'm not sure if we want to meld install & upgrade steps. Users upgrading from v1.0.0/config/v1.0/aws-k8s-cni-calico.yaml  will lose networkpolicy enforcement until they restart all pods.